frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tests.tsx

```
@@ -38,24 +38,26 @@ describe("Advanced settings — Activity & data retention", () => {
-    expect(screen.getByLabelText(/Disable vulnerabilities/i)).not.toBeChecked();
+    expect(screen.getByLabelText(/Disable hosts online historical reporting/i)).not.toBeChecked();
+    expect(screen.getByLabelText(/Disable vulnerability exposure historical reporting/i)).not.toBeChecked();
   });
 
   it("starts with the checkbox checked when collection is disabled in config", () => {
     renderAdvanced({
       historicalData: { uptime: false, vulnerabilities: true },
     });
     expect(screen.getByLabelText(/Disable hosts online/i)).toBeChecked();
-    expect(screen.getByLabelText(/Disable vulnerabilities/i)).not.toBeChecked();
+    expect(screen.getByLabelText(/Disable vulnerability exposure historical reporting/i)).not.toBeChecked();
```

ESLint

[error] Replace `screen.getByLabelText(/Disable·vulnerability·exposure·historical·reporting/i)` with `⏎······screen.getByLabelText(⏎········/Disable·vulnerability·exposure·historical·reporting/i⏎······)⏎····`

(prettier/prettier)